package.json

```diff
@@ -22,7 +22,16 @@
     "build-production-client": "NODE_ENV=production BABEL_ENV=production yarn build-client",
     "build-production": "yarn distclean && yarn && gulp languages:extract && yarn build-production-client",
     "build-languages": "gulp languages",
+    "build-with-docker": "docker run  --rm -it -v $(pwd):/jetpack -w /jetpack --name jetpack-builder oskosk/jetpack-builder bash -c 'yarn build'",
+    "build-languages-with-docker": "docker run --rm -it -v $(pwd):/jetpack -w /jetpack --name jetpack-builder oskosk/jetpack-builder bash -c 'yarn build-languages'",
+    "build-production-with-docker": "docker run  --rm -it --env NODE_ENV=production -v $(pwd):/jetpack -w /jetpack --name jetpack-builder oskosk/jetpack-builder bash -c 'yarn build'",
+    "docker:mysql-data": "[ -d .mysql-data ] || mkdir .mysql-data",
+    "docker:run": "yarn docker:mysql-data && docker-compose up -d",
+    "docker:start": "yarn docker:run",
+    "docker:stop": "docker-compose stop",
+    "docker:sh": "docker exec -it jetpack_wordpress bash",
     "lint": "eslint --ext .js --ext .jsx ./*.js _inc/client -c .eslintrc",
+    "watch-with-docker": "docker run  --rm -it -v $(pwd):/jetpack -w /jetpack --name jetpack-builder oskosk/jetpack-builder bash -c 'yarn watch'",
     "test-client": "NODE_ENV=test NODE_PATH=tests:_inc/client:node_modules/@automattic/dops-components/client tests/runner.js",
```

Comment:
Another command to add: `docker-compose down --rmi all -v` which will basically delete and clean up all the containers, images, and networks stored. Effectively a "docker:clean".